main.tf

```diff
@@ -2,111 +2,183 @@
-  count = var.create ? 1 : 0
-
-  name = var.cluster_name
-
-  dynamic "configuration" {
-    for_each = try([var.cluster_configuration], [])
-
-    content {
-      dynamic "execute_command_configuration" {
-        for_each = try([configuration.value.execute_command_configuration], [{}])
-
-        content {
-          kms_key_id = try(execute_command_configuration.value.kms_key_id, null)
-          logging    = try(execute_command_configuration.value.logging, "DEFAULT")
-
-          dynamic "log_configuration" {
-            for_each = try([execute_command_configuration.value.log_configuration], [])
-
-            content {
-              cloud_watch_encryption_enabled = try(log_configuration.value.cloud_watch_encryption_enabled, null)
-              cloud_watch_log_group_name     = try(log_configuration.value.cloud_watch_log_group_name, null)
-              s3_bucket_name                 = try(log_configuration.value.s3_bucket_name, null)
-              s3_bucket_encryption_enabled   = try(log_configuration.value.s3_bucket_encryption_enabled, null)
-              s3_key_prefix                  = try(log_configuration.value.s3_key_prefix, null)
-            }
-          }
-        }
-      }
-    }
-  }
-
-  dynamic "setting" {
-    for_each = [var.cluster_settings]
-
-    content {
-      name  = setting.value.name
-      value = setting.value.value
-    }
-  }
-
-  tags = var.tags
+module "cluster" {
+  source = "./modules/cluster"
+
+  create = var.create
+
+  cluster_name                     = var.cluster_name
+  cluster_configuration            = var.cluster_configuration
+  cluster_settings                 = var.cluster_settings
+  cluster_service_connect_defaults = var.cluster_service_connect_defaults
+
+  create_cloudwatch_log_group            = var.create_cloudwatch_log_group
+  cloudwatch_log_group_retention_in_days = var.cloudwatch_log_group_retention_in_days
+  cloudwatch_log_group_kms_key_id        = var.cloudwatch_log_group_kms_key_id
+  cloudwatch_log_group_tags              = var.cloudwatch_log_group_tags
+
+  default_capacity_provider_use_fargate = var.default_capacity_provider_use_fargate
+  fargate_capacity_providers            = var.fargate_capacity_providers
+  autoscaling_capacity_providers        = var.autoscaling_capacity_providers
+
+  create_task_exec_iam_role               = var.create_task_exec_iam_role
+  task_exec_iam_role_name                 = var.task_exec_iam_role_name
+  task_exec_iam_role_use_name_prefix      = var.task_exec_iam_role_use_name_prefix
+  task_exec_iam_role_path                 = var.task_exec_iam_role_path
+  task_exec_iam_role_description          = var.task_exec_iam_role_description
+  task_exec_iam_role_permissions_boundary = var.task_exec_iam_role_permissions_boundary
+  task_exec_iam_role_tags                 = var.task_exec_iam_role_tags
+  task_exec_iam_role_policies             = var.task_exec_iam_role_policies
+  create_task_exec_policy                 = var.create_task_exec_policy
```

Comment:
Maybe add a new line before `create_...` and comments like you do in the `service` module to separate blocks.

Reply:
Updated in https://github.com/terraform-aws-modules/terraform-aws-ecs/pull/76/commits/c6fef3b1bb52c658ce3cd0a120a38c6e17bef170